simulation.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import geopy
+from geopy.distance import great_circle
+import pyipinfodb
+import replay_log
+
+API_KEY = 'f096f204a09d53c278c457d8de90ba1aca8d9ede50a6399849045954a4e23535'
+SOUCE_INDEX =  2
+DESTINATION_INDEX = 3
+
+# This function may be replaced by cached version
+# calculate distance
+def get_distance(ip_addr1, ip_addr2):
```

Comment:
Can you move this to another file say `simulation_util.py` so that other components can also use this method.